### PR TITLE
Linux/mac fix for setup

### PR DESCRIPTION
**Pull request by mallewellyn**
This definitely needs looking at/tested, especially on Mac. I tested on Linux and it seems to work and removes igraph/minfi issues.

To explain PR:
- Because the second link in #160 requires a mixture of R and bash to install dependencies, it could be easier to use the first link and require all system dependencies are installed. Checked the latter and the dependencies don't seem to take too long to install. Could replace the Linux caveat (currently about igraph) with something similar to the code I've proposed.
- Tested on Ubuntu without any problems and seems to resolve igraph issue. I actually don't know that the Ubuntu code can be used on Mac as stated in this new version so will need to be tested/checked (my understanding is that the Mac terminal uses bash?).

### CODE REVIEW

File: setup.md [hunk 1]
```diff
@@ -9,8 +9,10 @@ to set up for the lessons. This should take about an hour to run, depending on
 the speed of your computer, your internet connection, and any packages you have
 installed already. You'll need to install R 4.0 or later.
 
-The following code will download the data and install the libraries
-used in the current version of this lesson:
+On Linux and Mac systems, R package installation often requires additional system dependencies. If you are a Linux or Mac user, to ensure that you can download packages using the code below, first run the terminal commands for your distribution [here](https://docs.posit.co/connect/admin/r/dependencies/). For Mac systems, you can use the Ubuntu code. Note that you will need to use root access (sudo) to install the system dependencies. Windows users can skip this step.
```
Comment (alanocallaghan) [inline setup.md:12]: Can mac users actually run apt install...?

Comment (alanocallaghan) [reply setup.md:12]: I think it'd be homebrew for a mac package manager, but ideally we'd get them just using binaries as well...

Comment (ailithewing) [reply setup.md:12]: I think as comment to say to just use binaries is ideal.

Comment (alanocallaghan) [inline setup.md:12]:
```suggestion
R usually enables package downloads using pre-built binaries. Some times, this is not possible,
particularly on Linux and Mac systems. In this case, R package installation often requires additional
system dependencies.
If you are a Linux, to ensure that you can download packages using the code below, first run the terminal 
commands for your distribution [here](https://docs.posit.co/connect/admin/r/dependencies/).
Note that you will need to use root access (sudo) to install the system dependencies.
Mac users may need to use homebrew, and Windows users may need to install [RTools](https://cran.r-project.org/bin/windows/Rtools/).
Ideally, installing packages will proceed without error and you can ignore these steps, but this isn't always the case.

Previous learners have reported issues with **`igraph`**. Installation instructions for this package can be found on [https://r.igraph.org/](https://r.igraph.org/),
```

Comment (ailithewing) [reply setup.md:12]: Looks good. Small typo on line 14, should be 'If you are a Linux user,'

**Pull request merged**